app/Filament/Admin/Resources/AccountResource/RelationManagers/RetentionChecksRelationManager.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Filament\Admin\Resources\AccountResource\RelationManagers;
+
+use App\Models\Training\WaitingList\WaitingListRetentionCheck;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Tables;
+use Filament\Tables\Table;
+
+class RetentionChecksRelationManager extends RelationManager
+{
+    protected static string $relationship = 'retentionChecks';
+
+    protected static ?string $recordTitleAttribute = 'id';
+
+    protected static ?string $title = 'Retention Checks';
+
+    protected function getLogActionName(): string
+    {
+        return 'ViewRetentionChecks';
+    }
```

Comment:
You only need this if you're using the `LogRelationAccess` trait. 

Example here: https://github.com/VATSIM-UK/core/blob/8fdab6140361bbbeb72d76714ba21607a79bdf58/app/Filament/Admin/Resources/AccountResource/RelationManagers/NotesRelationManager.php#L19C9-L19C26